src/strings/en-us.json

```diff
@@ -822,8 +824,8 @@
     "LabelSaveLocalMetadataHelp": "Saving artwork into media folders will put them in a place where they can be easily edited.",
     "LabelScheduledTaskLastRan": "Last ran {0}, taking {1}.",
     "LabelScreensaver": "Screensaver:",
-    "EnableFastImageFadeIn": "Fast image fade-in",
-    "EnableFastImageFadeInHelp": "Enable faster fade-in animation for loaded images",
+    "EnableFastImageFadeIn": "Image Fade Animations",
+    "EnableFastImageFadeInHelp": "Show posters and other images with a fade animation once they load.",
```

Comment:
My understanding is this doesn’t disable the animation entirely, just makes it faster. Perhaps that should be clarified here?